python/src/usecases/fireSatellite_function.py

```diff
@@ -351,7 +351,7 @@ def power(self, inputs):
         Ptot = P_ACS + P_other
 
         # power production capability at beginning of life
-        P_BOL = eta * Fs * Id * m.cos(theta)
+        P_BOL = eta * Fs * Id * m.cos(self.i)
```

Comment:
theta is not used anymore and linter fails

Reply:
Yes, the modification has been made to remove theta